src/He.PipelineAssessment.UI/Features/Assessment/AssessmentList/AssessmentListCommandHandler.cs

```diff
@@ -34,23 +34,14 @@ public async Task<AssessmentListData> Handle(AssessmentListCommand request, Canc
         private AssessmentListData GetAssessmentListFromResults(List<Assessment> listOfAssessments)
         {
             AssessmentListData assessmentLandingPageData = new AssessmentListData();
-            Random random = new Random(); //Not needed in anything but the dummy data.  TODO:  Remove when no longer needed;
             foreach (var assessment in listOfAssessments)
             {
                 var assessmentDisplay = new AssessmentDisplay(assessment);
-                assessmentDisplay.DateCreated = GetDateCreated(random);
+                assessmentDisplay.DateCreated = DateTime.UtcNow;
```

Comment:
it would be ideal to use DateTime provider here, then the DateCreated field can be unit tested properly too

Reply:
as this is only dealing with the dummy data for now, it's ok to refactor it later and create a shared DateTimeProvider class (which currently lives in Elsa Server)